index.html

```diff
@@ -3,22 +3,27 @@
 <head>
   <title>Astronomsko Društvo Omiš</title>
   <meta charset="UTF-8">
+  <link rel="stylesheet" href="public/assets/icons/css/font-awesome.min.css">
   <link rel="stylesheet" href="public/style.css">
 </head>
 
 <body>
   <header>
     <div class="brand">
-      <img src="" alt="">
-      <h1>Adomis</h1>
+      <a href="#">
```

Comment:
Since we have fixed header, it'd be best if this would link to the top of the page.